docs/migration.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: index
+title: Contributing
+navigation_weight: 7
+---
+# Migration from sw-framework
+
+[Intro]
+
+## Command-Line Interface (sw-precache)
+
+Just like `sw-precache`, Workbox allows you generate a service worker using the command line using [`sw-cli`](sw-cli). There are a few differences between the two.
+
+* Information that was previously passed as command line arguments is now gathered by a wizard that provides the option of saving your choices to a config file (`sw-cli-config.json`).
```

Comment:
`sw-cli` takes in command line flags as well as supporting a config file. `sw-precache` supported both, too.

`sw-precache` didn't have a wizard for initial configuration, though.